index.html

```diff
@@ -37,6 +37,148 @@ <h1>Hey there. I'm <br />Subrat </h1>
+      <div class="card">
+        <div class="cardImage">
 
+        </div>
+        <div class="cardBody">
+          <h2 class="cardHeading">Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="cardTags">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+          </ul>
+          <a href="#" class="cardAction">See Project</a>
+        </div>
+      </div>
+      <div class="card">
+        <div class="cardImage">
+
+        </div>
+        <div class="cardBody">
+          <h2 class="cardHeading">Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="cardTags">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+          </ul>
+          <a href="#" class="cardAction">See Project</a>
+        </div>
+      </div>      
+      <div class="card">
+        <div class="cardImage">
+
+        </div>
+        <div class="cardBody">
+          <h2 class="cardHeading">Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="cardTags">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+          </ul>
+          <a href="#" class="cardAction">See Project</a>
+        </div>
+      </div> 
+      <div class="card">
+        <div class="cardImage">
+
+        </div>
+        <div class="cardBody">
+          <h2 class="cardHeading">Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="cardTags">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+          </ul>
+          <a href="#" class="cardAction">See Project</a>
+        </div>
+      </div> 
+      <div class="card">
+        <div class="cardImage">
+
+        </div>
+        <div class="cardBody">
+          <h2 class="cardHeading">Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="cardTags">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+          </ul>
+          <a href="#" class="cardAction">See Project</a>
+        </div>
+      </div> 
+      <div class="card">
+        <div class="cardImage">
+
+        </div>
+        <div class="cardBody">
+          <h2 class="cardHeading">Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="cardTags">
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>JavScript</li>
+            <li>html</li>
+          </ul>
+          <a href="#" class="cardAction">See Project</a>
+        </div>
+      </div>                         
+    </section>
+    <section id="aboutMself">
+      <h2 class="cardHeading">About Me</h2>
+      <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>

```

Comment:
- [ ] I think it would be nice if you stick to the design template for the **_About Me_** Header. Kindly use the same font-size and margins for the header
- [ ] I would like to advise that you make the background-image touch the edges of your project using the template design as a guideline. 

### Your Design
![image](https://user-images.githubusercontent.com/77106962/143511077-cf5cdd37-102f-4aa9-a87f-55fef8318434.png)

### Figma's Design
![image](https://user-images.githubusercontent.com/77106962/143511148-d46c663a-3838-49a5-8b48-678c24db128e.png)

Reply:
updated